### DISCUSSION TITLE
Display tool tip of an alternate name for column header

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1257] [Data Modeling]
Are there any other column/measure properties that can be exposed and used on a report?
I spend way to much time renaming my measures when i am in a report.
ie to prevent wordwrap and wasting space, i will change the measure name Total Orders to Orders or Average Sales to Avg Sales or aSales.
It would be nice to choose which version of the name you want to use.

It would also be grand to hover over a column header and display a tool tip that is the description property.

Instead I waste time constructing mapping table and try exposing them through info pop up button.

I am really surprised more people don't have this issue.
It is rare that you can name a measure correctly the first time and to fit different scenarios
If you use SSAS than any name you make in the model in the future will break any report that is attached to it

**Comment by mlonsk** [comment]
Hi @TinyTimothy2 
I am not aware of any way to do what you propose, I believe this is just the way the SSAS engine works together with Power BI. There might be Power BI specific workarounds, but I have at least not heard of any.
